tests/contrib/pylons/test_pylons.py

```diff
@@ -674,11 +676,60 @@ def test_pylons_body_xml(self):
             assert span
             assert span["mytestingbody_key"] == "mytestingbody_value"
 
-    def test_pylons_body_xml_attack(self):
+    def test_pylons_body_json_unicode_decode_error_charset(self):
+        # Regression test, if request.charset returns None, a TypeError is raised
```

Comment:
I don't see where in the test code this `TypeError` is checked for. What am I missing?

Reply:
You are right, it's not checked for: if a regression happens, the TypeError will be raised and the test will fail.